spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -90,47 +95,53 @@ def create_workspace_csv(workspaces, client_id_column)
     prod_connection_parameters = {
       username: @config[:username],
       password: @config[:password],
-      server: "https://#{@config[:prod_server]}"
+      server: "https://#{@config[:prod_server]}",
+      verify_ssl: false
     }
     @prod_rest_client = GoodData.connect(prod_connection_parameters)
     @prod_ads = ConfigurationHelper.create_development_datawarehouse(client: @prod_rest_client,
                                                                      title: 'Production ADS',
                                                                      auth_token: @config[:vertica_prod_token])
-
-    @prod_output_stage_project = ConfigurationHelper.create_output_stage_project(
-      @prod_rest_client,
-      @suffix,
-      @prod_ads,
-      @config[:prod_token],
-      @config[:environment]
-    )
-    production_output_stage_uri = @prod_output_stage_project.add.output_stage.data['schema']
+    unless GoodData::Environment::VCR_ON
```

Comment:
The output stage thing produces a lot of `if`s. Consider testing the output stage in a separate "live" test. It might make things simpler.